spec/capybara/01_no_data_spec.rb

```diff
@@ -16,8 +16,9 @@
   it "Check All Entities" do
     visit '/entities_all'
 
-    content = [ 'Entities', 'failing out of 1' ]
+    content = [ 'Entities', 'failing out of' ]
```

Comment:
SSH and HTTP on localhost aren't open on these containers, so these checks were failing.